doc/examples/numpy_operations/plot_view_as_blocks.py

```diff
@@ -3,11 +3,11 @@
 Block views on images/arrays
 ============================
 
-This example illustrates the use of `view_as_blocks` from
-`skimage.util.shape`.  Block views can be incredibly useful when one
+This example illustrates the use of ``view_as_blocks`` from
+:py:func:`skimage.util.shape`.  Block views can be incredibly useful when one
```

Comment:
from :py:func:`skimage.util` (also, the imports at the top of the file should be modified so that we import from ``util`` and not ``util.shape``. I know this was not introduced by your PR, but it would be an improvement!